test/unit/loading-twice-iframe.html

```diff
@@ -0,0 +1,22 @@
+<html>
+<head>
+</head>
+<body>
+  <script>
+  window.addEventListener('message', () => {
+    const relativeLink = document.createElement('link');
+    relativeLink.href = '../../polymer.html';
+    relativeLink.rel = 'import';
+    document.body.appendChild(relativeLink);
+
+    const absoluteLink = document.createElement('link');
+    absoluteLink.href = '/polymer.html';
```

Comment:
Let's load this with a query string so we can avoid an abs url since that puts constraints on serving.

Reply:
It seems that, while query parameters can be used for cache busting, the browser will not load the same link twice even if the query parameters are different :sob: